specification.md

```diff
@@ -1059,6 +1061,140 @@ Note that the referenced function definition type in this case must be `rest` (d
 
 For more information about functions, reference the [Functions definitions](#Function-Definition) section.
 
+#### Using functions for HTTP Service Invocations
+
+For a few reasons, workflow designers might see necessary making straight HTTP invocations to other systems, such as:
+
+1. Target service might not expose an OpenAPI specification descriptor;
+2. Target service might not implement RESTful interfaces;
+3. The use case requires a simple HTTP request invocation.
+
+The specification supports describing HTTP invocations via a simplified interface in the [functions definition](#Function-Definition).
+
+The HTTP invocation interface can be described in the `operation` attribute of the function definition, as shown in the example:
+
+```json
+{
+"functions": [
+  {
+    "name": "queryUserById",
+    "operation": "GET#/users/{id}",
+    "type": "http"
+  }
+]
+}
+```
+
+The `operation` attribute is composed by a string separated by the hashtag symbol (`#`):
+
+* The HTTP method used by the HTTP invocation. Can be any [valid HTTP method](https://www.rfc-editor.org/rfc/rfc9110.html#name-method-definitions) such as `GET`, `POST`, `PUT`, etc.
```

Comment:
[RFC 9110](https://www.rfc-editor.org/rfc/rfc9110.html#name-methods) defines only "standardized methods that are commonly used", but HTTP supports many more than that: https://www.iana.org/assignments/http-methods/http-methods.xhtml#methods

Reply:
+100! 
i think we are opening a can of worms with this honestly unless we do add all these methods.
with this and all other comments from @gibson042 just confirms that maybe we really just need to allow runtimes to define this via extensions and provide a simple example only (showing that http is doable if wanted) and not get this "monster" into spec....

Reply:
@tsurdilo I disagree. Those methods are slmost never used, if ever. We could just say that they are all supported but that for the sake of brevity only common ones are documented.

@gibson042 I would personally fo the following:

`{method}#{url including scheme and host}`

Reply:
my problem is not i dont like the functionality its that we can already do this two ways via custom function type and expression function types. so if this does not from user perspective give some ability even if only limited like u said with grpc then its just adding more confusion than it solves

Reply:
and iirc we added custom function type before for just this :) 
and to add with this we are sctually adding a custom custom type cause not depending on a specification